src/libs/actions/Wallet.js

```diff
@@ -96,6 +96,15 @@ function setWalletShouldShowFailedKYC(shouldShowFailedKYC) {
     Onyx.merge(ONYXKEYS.USER_WALLET, {shouldShowFailedKYC});
 }
 
+/**
+ * Save the ID of the chat whose IOU triggered showing the KYC wall.
+ *
+ * @param {Number} chatReportID
+ */
+function setKYCWallSourceChatReportID(chatReportID) {
+    Onyx.merge(ONYXKEYS.USER_WALLET, {chatReportID});
```

Comment:
Do you think that we should set the chatReportID in the WALLET_TERMS onyx key instead? Since it's related to the terms step vs. the user's wallet.